znai-reactjs/src/doc-elements/jupyter/JupyterTextCell.js

```diff
@@ -17,11 +17,22 @@
 import React from 'react'
 
 const JupyterTextCell = ({text, elementsLibrary}) => {
+    const lines = convertToLines(text)
+
     return (
         <div className="jupyter-cell jupyter-text content-block">
-            <elementsLibrary.CliOutput lines={text.split('\n')}/>
+            <elementsLibrary.CliOutput lines={lines}/>
         </div>
     )
 }
 
+function convertToLines(text) {
+    const lines = text.split('\n')
+    if (lines.length > 0 && lines[lines.length - 1].length === 0) {
```

Comment:
Don't you need to lines.pop() for all of empty lines? For example, don't you need to make `xyx\n\n\n\n\n` to `xyz'?

Reply:
You are right, I was handling only natural last line line ending, but it makes sense to trim it fully. Updated.

Reply:
Do you still need `convertToLines()` to handle a natural last line ending?

Reply:
duh... changed :)